cscs-checks/apps/tensorflow/keras_check.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+imagenet_tar = '/apps/common/UES/reframe/resources/training/imagenet.tar'
```

Comment:
this is only about 10% of the actual imagenet dataset
(I don't know if it's necessary to add it to reframe, though)
Can we keep this PR just about PyTorch, please?